src/main/java/org/springframework/cloud/deployer/spi/kubernetes/DefaultContainerFactory.java

```diff
@@ -137,7 +157,53 @@ protected Probe createProbe(Integer externalPort, String endpoint, int timeout,
 		return cmdArgs;
 	}
 
-	/**
+    /**
+     * The list represents a single command with many arguments.
+     *
+     * @param request AppDeploymentRequest - used to gather application overridden
+     * container command
+     * @return a list of strings that represents the command and any arguments for that command
+     */
+    private List<String> getContainerCommand(AppDeploymentRequest request) {
+        List<String> containerCommandList = new ArrayList<>();
+        String containerCommand = request.getDeploymentProperties()
+                .get("spring.cloud.deployer.kubernetes.containerCommand");
+        if (containerCommand != null) {
+			logger.info("Building container command from AppDeploymentRequest: "
```

Comment:
All these should not be logged at info level, this is too broad. Should be debug, if not trace